plugins/regmap/src/xmlfilemanager.cpp

```diff
@@ -211,13 +211,13 @@ BitFieldModel *XmlFileManager::getBitField(QDomElement bitField)
 
 		return new BitFieldModel(
 			name, bitField.firstChildElement("Access").firstChild().toText().data(),
-			bitField.firstChildElement("DefaultValue").firstChild().toText().data().toInt(), description,
+			bitField.firstChildElement("DefaultValue").firstChild().toText().data().toUInt(), description,
```

Comment:
Create a conversion function from QString to uint32_t and use it throughout the regmap where needed.